examples/en2gr-runtime.cpp

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "../../tools/loader/TextTranslator.h"
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Importer/Caffe2ModelLoader.h"
+#include "glow/Partitioner/Partitioner.h"
+#include "glow/Runtime/HostManager/HostManager.h"
+#include "glow/Runtime/RuntimeTypes.h"
+#include "glow/Support/Error.h"
+#include "llvm/Support/FileSystem.h"
+
+#include <chrono>
+#include <future>
+
+using namespace glow;
+using namespace glow::runtime;
+
+namespace {
+llvm::cl::OptionCategory category("en2gr-runtime Options");
+llvm::cl::opt<unsigned> numDevices("numDevices",
+                                   llvm::cl::desc("Number of Devices to use"),
+                                   llvm::cl::init(5), llvm::cl::value_desc("N"),
+                                   llvm::cl::cat(category));
+} // namespace
+
+/// Builds a simple graph, returns back input var and save node through refs.
```

Comment:
it's hard to see how it returns back input var and save node ..

Reply:
updated the comment.